spec/plans/provision.fmf

```diff
@@ -171,7 +176,17 @@ example: |
 
         # Enable regular expression search
         cpu:
-            model_name: =~ .*AMD.*
+            model-name: =~ .*AMD.*
+
+        # Network interface
+        network:
+          - type: eth
+
+        # Features related to virtualization
+        virtualization:
+            is-supported: true
+            is-virtualized: false
+            hypervisor: kvm
```

Comment:
Nitpicking, but: if the `is-virtualized` is untrue, I'd expect a hypervisor to be `null` or `none` or something similar - `kvm` practically mandates the machine is virtualized :)

How about switching values of `is-supported` and `is-virtualized`?

```yaml
        # Features related to virtualization
        virtualization:
            is-supported: false
            is-virtualized: true
            hypervisor: kvm
```

This would actually be correct for many OpenStack tenants out there, if the setup disallows nested virtualization :)

Reply:
Right you are, this is more real-life. Changed.